codecov.yml

```diff
@@ -0,0 +1,6 @@
+coverage:
+  status:
+    project:
+      default:
+        # basic
+        threshold: 0.01%
```

Comment:
I'm not a fan of setting this, but we have in the past used admin privileges to force merge a PR when something like this occurs.

I would prefer this deleted and it force merged. If you don't have access you can ask @timbellnz for permission, or I should be able to merge it.

Reply:
Cool, that makes sense